src/compile/data/aggregate.ts

```diff
@@ -11,11 +12,9 @@ function addDimension(dims: {[field: string]: boolean}, fieldDef: FieldDef) {
     dims[field(fieldDef, {binSuffix: 'start'})] = true;
     dims[field(fieldDef, {binSuffix: 'end'})] = true;
 
-    // const scale = model.scale(channel);
-    // if (scaleType(scale, fieldDef, channel, model.mark()) === ScaleType.ORDINAL) {
-    // also produce bin_range if the binned field use ordinal scale
-    dims[field(fieldDef, {binSuffix: 'range'})] = true;
-    // }
+    if (fieldDef.type === ORDINAL) {
```

Comment:
They are not equivalent, this can be binned ordinal or temporal that uses an ordinal scale.  
Given you ask this question, I suspect that this fix is incorrect.  
Let me take a look.